assets/base.css

```diff
@@ -1761,8 +1778,7 @@ input[type='checkbox'] {
   color: rgba(var(--color-foreground));
   border-radius: var(--inputs-radius);
   position: relative;
-  height: 4.5rem;
-  width: calc(14rem / var(--font-body-scale));
+  width: calc(16rem / var(--font-body-scale));
```

Comment:
@melissaperreault To avoid this: https://screenshot.click/22-01-l078m-brnjt.mp4 , I updated the calculation to use 16 instead of 14. I see this was updated in @KaichenWang 's PR here: https://github.com/Shopify/dawn/commit/fe3de560f4c3fef7e35e18e0d9fc1515cdd99d53#diff-00b03bd62b155daa89082ba52094c182e46422a9e28cd3ec601b3a2291b52718R1646

Kai, since you worked on this change do you mind confirming?